prog/plugin/main.go

```diff
@@ -168,3 +170,19 @@ func createNetwork(dockerClient *docker.Client, networkName, address string) err
 	}
 	return nil
 }
+
+func listenAndServeStatus(socket string) {
+	if err := os.Remove(socket); err != nil && !os.IsNotExist(err) {
+		Log.Fatalf("Error removing existing status socket: %s", err)
+	}
+	listener, err := net.Listen("unix", socket)
+	if err != nil {
+		Log.Fatalf("ListenAndServeStatus failed: %s", err)
+	}
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "ok")
+	})
+	if err := (&http.Server{Handler: handler}).Serve(listener); err != nil {
+		Log.Fatalf("ListenAndServeStatus failed: %s", err)
+	}
+}
```

Comment:
this is c&p from the proxy. And the socket removal+listening bit is also duplicated in the plugin. And we really don't want those `Fatalf`, at least not in the first part.